react/features/base/conference/middleware.any.js

```diff
@@ -155,6 +169,7 @@ function _conferenceFailed({ dispatch, getState }, next, action) {
             window.removeEventListener('beforeunload', beforeUnloadHandler);
             beforeUnloadHandler = undefined;
         }
+        enableForcedReload && error?.name === JitsiConferenceErrors.CONFERENCE_RESTARTED && dispatch(reloadNow());
```

Comment:
The one inside the `if` condition is for the web version and it returns the next dispatch function from there. The second one outside of `if` is for the mobile case. The conference.leave function for web is called from conference.js

Reply:
OK. So maybe you can do:
```
if (typeof APP !== 'undefined') {
        if (typeof beforeUnloadHandler !== 'undefined') {
            window.removeEventListener('beforeunload', beforeUnloadHandler);	        
            beforeUnloadHandler = undefined;
        }	        
} else { // mobile
        !error.recoverable
        && conference
        && conference.leave().catch(reason => {
            // Even though we don't care too much about the failure, it may be	
            // good to know that it happen, so log it (on the info level).
            logger.info('JitsiConference.leave() rejected with:', reason);
        });
}

enableForcedReload && error?.name === JitsiConferenceErrors.CONFERENCE_RESTARTED && dispatch(reloadNow());

return result;
```

Isn't that look better without copy/pasted line? WDYT?